CONTRIBUTING.md

```diff
@@ -19,6 +19,18 @@ It is expected that all maintainers are watching those categories to allow us to
 
 When releasing a new package into a distribution, please follow the naming guidelines set out in the [ROS REP 144: ROS Package Naming](https://www.ros.org/reps/rep-0144.html).
 
+### Guidelines for Package Contents
+
+When releasing a new package into a ROS distribution, the reviewers will check that the package has the following characteristics:
+
+* The source repository must be publicly accessible.
+* The license should be one of the open-source ones listed in [https://opensource.org/licenses](https://opensource.org/licenses) (preferably one of the "Popular licenses").
+* The license must be reflected in the package.xml file of all sub-packages in the repository.
+* A LICENSE file exists in the source repository.  This is because the short identifiers in the package.xml file aren't specific enough to unambiguously identify the license.
+* The source repository should contain one or more ROS packages (meaning they have a `package.xml` in the source repository). Packages that are not ROS packages can be accepted, but they are rare and require special handling in the release repository.
```

Comment:
nit: "package.xml" is styled differently in each sentence

Reply:
Fixed in 2c09946.